datasource/admin.py

```diff
@@ -229,6 +229,32 @@ class SwitchitAdmin(DatasourceAdmin, admin.ModelAdmin):
 class UsnicAdmin(DatasourceAdmin, admin.ModelAdmin):
     form = CountriesWidgetOverrideForm
 
+    actions = ['add_to_Brands']
+
+    @admin.action(description='Create new related Brand and copy data')
+    def add_to_Brands(self, request, queryset):
+        self.message_user(request, 'Successfully added new brand', messages.SUCCESS)
```

Comment:
The success message should probably go to the bottom, to make sure it only displays if everything went well?